### DISCUSSION TITLE
Feature Request: Removing Organisation from Windows + Edge

### DISCUSSION THREAD

**Discussion by Kyle15Bezz** [ChrisTitusTech/winutil#847] [Ideas]
Would it be possible to add a button which removes the Organisation that is controlling some of the settings in Windows and in Microsoft Edge? As this is would be very helpful to when there is an unknown organisation controlling some settings and we have to remove them manually from registry or something like that. Adding this to your tool will be very helpful

**Comment by ChrisTitusTech** [comment]
--How to Remove Managed by your organization from Microsoft Edge--

- Launch the Windows Registry Editor. Press Win + R on the keyboard and type regedit in the Run box.
- Go to the following key: HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft.
- Locate the Edge folder and right-click it.
- Select Delete from the menu.
- Alternatively, you can delete all the parameters inside this folder.
- Now, open the following key: HKEY_CURRENT_USER\SOFTWARE\Policies\Microsoft\Edge
- Delete its contents as well.
- Restart the browser. The message should be gone now.

**Reply by MacReyhan** [reply]
@ChrisTitusTech And for Link to Phone .. same is appearing in settings and it is not working. How to fix that please help?

**Comment by ChrisTitusTech** [comment]
next update this will be the standard and you will no longer see this or need to do it after tweaks are finished.

**Reply by MikelZuniga** [reply]
> next update this will be the standard and you will no longer see this or need to do it after tweaks are finished.

Chris, i ahve this issue too, some settings now are not given acess in Coinfig, And the Notifications and other App from windows are disabled and i can not change that ones..
Would it be possible to add a button which removes the Organisation that is controlling some of the settings in Windows?
I need restore all to default and i cant!
Help please!
Thanks
